app/assets/stylesheets/_settings.scss

```diff
@@ -355,7 +355,7 @@ $fieldset-border: 1px solid $medium-gray;
 $fieldset-padding: rem-calc(20);
 $fieldset-margin: rem-calc(18 0);
 $legend-padding: rem-calc(0 3);
-$form-spacing: rem-calc(16);
+$form-spacing: rem-calc(10);
```

Comment:
Foundation forms are only used in the admin pages and not anywhere else, so it will not affect the whole website. `$form-spacing` reduces the spacing inside the form input fields.